Update broken link (#26396)
DataDog/documentation: content/en/developers/community/libraries.md

### Aptible

> The Aptible platform can deliver container logs and metrics to a Datadog account. [Consult the dedicated Aptible help center to learn how][25].

Author: ```suggestion
The Aptible platform can deliver container logs and metrics to a Datadog account. Consult the [Aptible help center][25] for details.
```
